backend/app/categories/controller.py

```diff
@@ -19,22 +19,22 @@ async def get_category_route(category_id: str,
                              db: AsyncIOMotorClient = Depends(get_database)) -> ItemsInResponse:
     """Get the items under a given category"""
         
-    loader = None
-    if category_id == CategoryEnum.movies.value:
-        loader = Movie
-    if category_id == CategoryEnum.anime.value:
-        loader = Anime
-    if category_id == CategoryEnum.music.value:
-        loader = Music
-    if category_id == CategoryEnum.shows.value:
-        loader = Show
-    if category_id == CategoryEnum.books.value:
-        loader = Book
-    if category_id in CategoryEnum.__members__:
+    if category_id not in CategoryEnum.__members__:
+        return ItemsInResponse(success=False, error=[f"Invalid category {category_id}"])
+    else:
```

Comment:
Don't need the `else` here, since there is a `return` statement in the `if` block